src/test/java/com/networknt/schema/LocaleTest.java

```diff
@@ -145,11 +145,13 @@ void encoding() {
         expected.put("vi","$: phải dài tối đa 5 ký tự");
         expected.put("zh_CN","$: 长度不得超过 5 个字符");
         expected.put("zh_TW","$: 長度不得超過 5 個字元");
+        expected.put("es","$: debe tener como máximo 5 caracteres");
 
         // In later JDK versions the numbers will be formatted
         Map<String, String> expectedAlternate = new HashMap<>();
         expectedAlternate.put("ar","$: يجب أن يكون طوله ٥ حرفًا على الأكثر");
         expectedAlternate.put("fa","$: باید حداکثر ۵ کاراکتر باشد");
+        expectedAlternate.put("es","$: debe tener como mÃ¡ximo 5 caracteres");

```

Comment:
What is this comparison for and how is this possible? The rest of the alternate comparisons is because later JDK versions perform mapping to arabic numerals.

Reply:
Note that for Java 8 the properties file must be encoded in ISO 8859-1. UTF-8 is only supported from Java 9 onwards. If the test fails for Java 8 but passes for the rest it means that your file is not encoded in ISO 8859-1.

Reply:
@justin-tay 
in the English text "Hello, how are you?" is encoded to ISO 8859-1 and then decoded back to a string. However, since ISO 8859-1 does not include characters specific to Spanish, the output will be the same as the input.

Reply:
If the encoding is correct then remove the expectedAlternate line and the test should pass. If not it's just not encoded correctly and is being corrupted to this `mÃ¡ximo` text. This means that anyone using Java 8 will see `debe tener como mÃ¡ximo 5 caracteres` as the translated text which is obviously not correct.

Reply:
@justin-tay do you prefer any tool for conversion. so that i can verify once more.

Reply:
I don't prefer any tool, it just need to be encoded such that Java 8 can load it properly. I just use Eclipse which will automatically convert text into ISO 8859-1 when pasted into a properties file as that is the default configuration in Eclipse for the Java Properties File content type.

Reply:
@justin-tay i tried the same i set my intelij editor to ISO 8859-1 i commited  now still java 8 is failing. can you please do me a favour for this

Reply:
@justin-tay can you please give me the converted file. its needed for us

Reply:
@justin-tay please let me any help.